Data Structures - Stacks and Queues/Stacks using Arrays.py

```diff
@@ -3,16 +3,31 @@ def __init__(self):
     self.arr = []
     self.length = 0
   
+  # return stack 
   def __str__(self):
-    return str(self.__dict__)
+    return str(self.arr)
```

Comment:
Change the statement to return str(self.__dict__), because I intended to print the array and the length of the array.